app/onboarding/page.tsx

```diff
@@ -56,74 +65,130 @@ interface ReviewPageProps {
 const StateSelection = ({
   selectedState,
   setSelectedState,
-}: StateSelectionProps) => {
+  onNext,
+}: StateSelectionProps & { onNext: () => void }) => {
   return (
-    <div>
-      <h2>Which state do you live in?</h2>
-      <select
-        value={selectedState}
-        onChange={e => setSelectedState(e.target.value)}
-      >
-        <option value="" disabled>
-          Select a state
-        </option>
-        {stateOptions.map(state => (
-          <option key={state.label} value={state.value}>
-            {state.label}
-          </option>
-        ))}
-      </select>
-    </div>
+    <OnboardingContainer>
+      <Flex $direction="column" $align="center">
+        <ProgressBar progress={2.6} />
+        <P3 $color={COLORS.shrub} style={{ marginTop: '2.5rem' }}>
+          QUESTION 1 OF 3
+        </P3>
```

Comment:
since the progress bar and the question number are shared between multiple pages, we should move it outside of these individual components and into the OnboardingFlow component to reduce redundancy

If we end up making onboarding a multi-page flow next sem, these would be good candidates of things to include in the layout.tsx

So the individual question components can primarily just be the Question Title + the RadioOptions 

we might even consider creating a general component `OnboardingQuestion` that can be used for all 3 questions